load-gen/src/main/java/io/smartcat/data/loader/api/Terminateable.java

```diff
@@ -0,0 +1,13 @@
+package io.smartcat.data.loader.api;
+
+/**
+ * Has ability to terminate itself.
+ */
+public interface Terminateable {
```

Comment:
So you are not going to switch to AutoCloseable which already provides the close method?